requirements.txt

```diff
@@ -4,4 +4,4 @@ pymysql
 geojson
 discord.py
 rich
-coloredlogs
\ No newline at end of file
+coloredlogs
```

Comment:
probably all of those needs to be fixed to a version. otherwise there would be an issue on installing it next year again e.g.